libs/blocks/section-metadata/section-metadata.js

```diff
@@ -1,4 +1,10 @@
 import { handleFocalpoint } from '../../utils/decorate.js';
+import { createTag, getConfig } from '../../utils/utils.js';
+import { decorateDefaultLinkAnalytics } from '../../martech/attributes.js';
+import { replaceKey } from '../../features/placeholders.js';
+
+const replacePlaceholder = async (key) => replaceKey(key, getConfig());
```

Comment:
I still don't quite understand why this isn't fetched from the Federated content, something like:
```suggestion
import { createTag, getConfig, getFedsPlaceholderConfig } from '../../utils/utils.js';
import { decorateDefaultLinkAnalytics } from '../../martech/attributes.js';
import { replaceKey } from '../../features/placeholders.js';

const replacePlaceholder = async (key) => replaceKey(key, getFedsPlaceholderConfig());
```
Trying it locally yields the expected results.